### PR TITLE
[I18n] Warn for illegal characters

### PR DESCRIPTION
**Pull request by RedGl0w**
This patch https://github.com/numworks/epsilon/issues/1173
## An example of logs :
```
I18N    apps/i18n.cpp
👍 (0x128077) is not a valid character
```
## How it works :
It just parse the code_points.h file, and check for each character if it is inside this file

### CODE REVIEW

**Comment by Ecco** [comment]
That's interesting, thank! Out of curiosity, in which scenario would this be useful in practice?

**Comment by RedGl0w** [comment]
> That's interesting, thank! Out of curiosity, in which scenario would this be useful in practice?

On omega we're doing a support for Hungarian, and we got wrong character, but we discovered only when I created and tested this PR

**Comment by RedGl0w** [comment]
> > That's interesting, thank! Out of curiosity, in which scenario would this be useful in practice?
> 
> On omega we're doing a support for Hungarian, and we got wrong character, but we discovered only when I created and tested this PR

Also, I just remembered when we added some physic constants, I tried the ^-1 character, and realized that there were no errors given, and I found that it was strange...

**Comment by RedGl0w** [comment]
Any news @Ecco ?

File: kandinsky/Makefile [hunk 1]
```diff
@@ -25,6 +25,10 @@ tests_src += $(addprefix kandinsky/test/,\
   rect.cpp\
 )
 
+code_points += $(addprefix kandinsky/fonts/, \
+  code_points.h \
```
Comment (EmilieNumworks) [inline kandinsky/Makefile:29]: `code_points = kandinsky/fonts/code_points.h`

**Comment by EmilieNumworks** [comment]
Good idea @RedGl0w. I'd like to merge this, could you please resolve the conflicts with the master branch? Could you also avoid the "Merge" commit. When possible, we would rather keep a "clean" memory.

**Comment by RedGl0w** [comment]
I'll recreate my PR, because I had to delete my repo, and so I can't commit anymore :/

**Pull request closed without merging**